extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -140,16 +126,16 @@ export class NetCoreTool extends ShellExecutionHelper {
 	}
 
 	/**
-	 * This function checks if the installed dotnet version is between minSupportedNetCoreVersion (inclusive) and maxSupportedNetCoreVersionCutoff (exclusive).
-	 * When maxSupportedNetCoreVersionCutoff is not set, the latest dotnet version is assumed to be supported and only the min version is checked.
+	 * This function checks if the installed dotnet version is atleast minSupportedNetCoreVersion.
+	 * Versions lower than minSupportedNetCoreVersion aren't supported for building projects.
 	 * Returns: True if installed dotnet version is supported, false otherwise.
 	 * 			Undefined if dotnet isn't installed.
 	 */
 	private async isNetCoreVersionSupported(): Promise<boolean | undefined> {
 		try {
 			const spawn = child_process.spawn;
 			let child: child_process.ChildProcessWithoutNullStreams;
-			let installState: netCoreInstallState = netCoreInstallState.netCoreVersionSupported;
+			let isSupported: boolean | undefined = undefined;
```

Comment:
I'd probably just default this to false - undefined isn't really something that should ever happen (and if it did we'd just treat it like false anyways...)